policy/modules/services/afs.if

```diff
@@ -54,7 +54,7 @@ interface(`afs_rw_cache',`
 	')
 
 	files_search_var($1)
-	allow $1 afs_cache_t:file { read write };
```

Comment:
Here `open` gets added, but the interface name does not mention inherited access

Reply:
I suspect in this case adding `open` is correct. My guess is that afs isn't used much so no one has actually hit this.

Reply:
The `open` access is expected, otherwise the interface should be named `afs_rw_inherited_cache_files` or similiar